From fee6899c704b2e6fbd404ffcae3055383a983a61 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Tue, 24 Mar 2026 12:44:06 +0000
Subject: [PATCH 2/3] remove persist.sys.usb.config=adb default from base.mk

adb is disabled by default on release builds; enabling it requires
the usual developer options + usb debugging toggle. debug builds
re-enable it via the debug tier.
---
 base.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/base.mk b/base.mk
index fda86c1..2462c9b 100644
--- a/base.mk
+++ b/base.mk
@@ -42,7 +42,6 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
 
 # Huawei HiSuite (also other OEM custom programs I guess) it's of no use in AOSP builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
-    persist.sys.usb.config=adb \
     ro.cust.cdrom=/dev/null
 
 # VNDK config files
-- 
2.53.0

